### PR TITLE
Update EIP-2255: Add myself as an author and move back to review

### PR DESCRIPTION
**Pull request by Pandapip1**
Will need review from either Dan Finlay (@danfinlay) or Erik Marks (@rekmarks)

Some more changes are necessary, but this is a reminder to self to do it.

### CODE REVIEW

File: EIPS/eip-2255.md [hunk 1]
```diff
@@ -1,42 +1,31 @@
...
-  <summary>Facebook Permissions</summary>
-  <img src="../assets/eip-2255/facebook_permissions.png" alt="Facebook Permissions" />
-</details>
+![Facebook Permissions][../assets/eip-2255/facebook_permissions.png]{loading=lazy}
 
-<details>
-  <summary>Log in With Apple</summary>
-  <img src="../assets/eip-2255/log_in_with_apple.jpeg" alt="Log in With Apple" />
-</details>
+![Log in With Apple][../assets/eip-2255/log_in_with_apple.jpeg]{loading=lazy}
```
Comment (Pandapip1) [inline EIPS/eip-2255.md:28]: @SamWilsn we've got a false positive!

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Review by danfinlay — APPROVED** [review]
Thanks for cleaning this up! These look fine to me.

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Pull request merged**